<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>600</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Appname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>460</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Version">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>460</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Developer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>460</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AboutQt">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>600</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Sourcecode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Source code</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
